blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpIntegrationTestBase.java

```diff
@@ -272,6 +279,16 @@ private HttpPost getHttpPost( String tenantId, String urlPath, String json ) thr
         return post;
     }
 
+    private HttpPost getHttpPost( String tenantId, String urlPath, String content, ContentType contentType  ) throws URISyntaxException {
```

Comment:
I think the first `getHttpPost` method should call the new one, so as to avoid duplication.